src/stylesheets/components/_card.scss

```diff
@@ -14,7 +22,6 @@
   .sds-card__header-mobile {
     color: #212121;
```

Comment:
Prefer to use the token value and mixin/utils to grab color, background color, margin, padding, etc.
(see https://designsystem.digital.gov/utilities/ for all available)